GoogleForCreators/web-stories-wp
# Hide tooltip while design menu settings are open

## Relevant Technical Choices

- I removed the auto-focus of the button when the menu is closed, because keyboard navigation doesn't work anyway, and it causes the tooltip to render in the wrong location sometimes (you cannot open the submenu and navigating in it using the keyboard - focus stays on the button when you open the submenu).

## User-facing changes

| Before | After |
|-|-|
| <img width="695" alt="Screen Shot 2022-12-20 at 22 41 19" src="https://user-images.githubusercontent.com/637548/208772392-5e9467bf-1e82-4df3-bd75-ade58e4fd5f4.png"> | <img width="695" alt="Screen Shot 2022-12-20 at 22 40 46" src="https://user-images.githubusercontent.com/637548/208772409-a2e3d706-04b4-4c52-9b0e-a499a929e660.png"> |

## Testing Instructions

This PR can be tested by following these steps:

1. Add an element to canvas
2. In the floating design menu, hover the settings button (⚙️)
3. Observe that a tooltip appears
4. Now click the button
5. Observe that the menu opens and the tooltip disappears and the tooltip cannot be made to appear while the settings menu is open
6. Close the menu (in any possible way)
7. Hover the settings button again and observe that the tooltip appears again

## Checklist




---

Fixes #12250
